fulp_modules/QM_wrestling/code/wrestling.dm

```diff
@@ -0,0 +1,455 @@
+/*
+ *	The contents of this file were originally licensed under CC-BY-NC-SA 3.0 as part of Goonstation(https://ss13.co).
+ *	However, /tg/station and derivative codebases have been granted the right to use this code under the terms of the AGPL.
+ *	The original authors are: cogwerks, pistoleer, spyguy, angriestibm, marquesas, and stuntwaffle.
+ */
+
+/// Defines the Martial Art - Move to fulp_defines.dm once said file exists -Willard
+#define MARTIALART_FULPWRESTLING "Quartermaster Wrestling"
+/// How many steps it takes to throw the target - Used for throwing
+#define WRESTLING_THROW_STEPS 28
+
+
+/datum/martial_art/fulpwrestling
+	name = "Quartermaster Wrestling"
+	id = MARTIALART_FULPWRESTLING
+	help_verb = /mob/living/proc/FulpWrestling_help
+	smashes_tables = TRUE
+	var/datum/action/fulpslam/slam = new/datum/action/fulpslam()
+	var/datum/action/fulpdrop/drop = new/datum/action/fulpdrop()
+	var/list/valid_areas = list(/area/quartermaster)
```

Comment:
are there any actual "/area/quartermaster"s in the game?

Reply:
Literally look at the code, Ath.
"/area/quartermaster" are all cargo bay and office.

Reply:
it was renamed to /area/cargo in the latest version of TG because """code improvement"""